examples/upbound-aws-provider/composite-resources/serverless-examples/microservice/README.md

```diff
@@ -0,0 +1,337 @@
+sns.notification.upbound.awsblueprints.io             XNotification     awsblueprints.io/v1alpha1   23m
+table.dynamodb.awsblueprints.io                       XDynamoDBTable    awsblueprints.io/v1alpha1   23m
+write-cloudwatch-metrics.iampolicy.awsblueprints.io   IAMPolicy         awsblueprints.io/v1alpha1   23m
+write-dynamodb.iampolicy.awsblueprints.io             IAMPolicy         awsblueprints.io/v1alpha1   23m
+write-firehose-s3.iampolicy.awsblueprints.io          IAMPolicy         awsblueprints.io/v1alpha1   23m
+write-firehose.iampolicy.awsblueprints.io             IAMPolicy         awsblueprints.io/v1alpha1   23m
+write-s3.iampolicy.awsblueprints.io                   IAMPolicy         awsblueprints.io/v1alpha1   23m
+write-sqs.iampolicy.awsblueprints.io                  IAMPolicy         awsblueprints.io/v1alpha1   23m
+```
+
+### Build Lambda function packages
+
+This example uses ZIP packaging of the Lambda functions. 
+ - Navigate to `./src/logic/` directory
+ - Create ZIP file `microservice-business-logic.zip` with the `lambda_function.py` in it:
+```shell
+zip microservice-business-logic.zip lambda_function.py
+```
+ - Navigate to `./src/authorizer/` directory
+ - Create ZIP file `microservice-authorizer.zip` with the `lambda_function.py` in it:
+```shell
+zip microservice-authorizer.zip lambda_function.py
+```
+ - Create Amazon S3 bucket
+ - Upload both ZIP files to the S3 bucket you just created
+
+### Update and apply the claim
+
+Make sure you are in the following directory:
+```shell
+cd examples/upbound-aws-provider/composite-resources/serverless-examples/microservice/
+```
+
+Set the AWS region in the claim with the ones used in the previous step “Build Lambda function packages” where the Lambda ZIP packages were uploaded to S3.
+```shell
+export AWS_REGION=<replace-with-aws-region> # example `us-east-1`
+export S3_BUCKET=<replace-with-s3-bucket-name> # example `my-crossplane-microservice-lambdas`
+```
+
+Change the default value for `CLAIM_NAME` with any name you choose.
+```shell
+export CLAIM_NAME=<replace-with-claim-name> # example `test-rest-api`
+```
+
+Run the below command to use the template file `microservice-claim-tmpl.yaml` in the `claim` folder to create the claim file with the variables `CLAIM_NAME`, `S3_BUCKET`, and `AWS_REGION` substituted.
+```shell
+envsubst < "claim/microservice-claim-tmpl.yaml" > "claim/microservice-claim.yaml"
+```
+
+Check that the claim populated with values. Update API name or description values in the claim if desired.
+```
+cat claim/microservice-claim.yaml
+```
+
+Apply the claim
+```shell
+kubectl apply -f claim/microservice-claim.yaml
+```
+
+Validate the claim
+```
+kubectl get microservices
+```
+
+Expected result (it might take sometime before READY=True)
+```
+NAME                 SYNCED   READY   CONNECTION-SECRET   AGE
+test-rest-api        True     True                        1m
+```
+The claim will create the following resources:
+
+```mermaid
```

Comment:
Although this is helpful in understanding the relationships of XRs and MRs, it's a bit too much when it's rendered. I think these are best left out for other UIs.

Reply:
Replaced with crossplane trace output.